core/tests/unit/ldml/keyboards/k_200_reorder_nod_Lana.xml

```diff
@@ -35,12 +35,12 @@
   <transforms type="simple">
     <transformGroup>
       <!-- from the spec: nod-Lana for ᩅᩫ᩶  -->
-      <reorder from="\u1A60" order="127" />
-      <reorder from="\u1A6B" order="42" />
-      <reorder from="[\u1A75-\u1A79]" order="55" />
-      <reorder before="\u1A6B" from="\u1A60\u1A45" order="10" />
-      <reorder before="\u1A6B[\u1A75-\u1A79]" from="\u1A60\u1A45" order="10" />
-      <reorder before="\u1A6B" from="\u1A60[\u1A75-\u1A79]\u1A45" order="10 55 10" />
+      <reorder from="\u{1A60}" order="127" />
```

Comment:
Are the specs already updated for these?

Reply:
the spec has long had `\u{...}` only, but the other form was allowed (it's ICU's native)